### PR TITLE
[#934] Parse (but don't map) API message response…

### PR DESCRIPTION
**Pull request by paulodiniz**
…content

### CODE REVIEW

File: backend/model/message/src/main/java/co/airy/model/message/MessageRepository.java [hunk 1]
```diff
@@ -29,5 +37,18 @@ public static String resolveContent(Message message, Map<String, String> metadat
...
+
+        try {
+            jsonNode = new ObjectMapper().readTree(resolvedContent);
+        } catch (JsonProcessingException e) {
+            ObjectMapper objectMapper = new ObjectMapper();
+            try {
+                jsonNode = objectMapper.readTree("{\"text\": \"" + content + "\", \"type\": \"text\"}");
+            } catch (JsonProcessingException ex) {
+                ex.printStackTrace();
+            }
```
Comment (chrismatix) [inline backend/model/message/src/main/java/co/airy/model/message/MessageRepository.java:49]: In the ticket we agreed on returning the content as is if we cannot parse it to a string so I would simply `return resolvedContent;` in the `catch` clause.

**Review by chrismatix — COMMENTED** [review]
The Typescript and doc changes are good 👍

File: backend/model/message/src/main/java/co/airy/model/message/MessageRepository.java (hunk 1 above)
Comment (paulodiniz) [reply backend/model/message/src/main/java/co/airy/model/message/MessageRepository.java:49]: We cannot because the return type of `content` is `JsonNode`

Comment (paulodiniz) [reply backend/model/message/src/main/java/co/airy/model/message/MessageRepository.java:50]: This discussion is now on https://github.com/airyhq/airy/issues/934#issuecomment-779759231

File: frontend/ui/src/pages/Inbox/ConversationListItem/index.tsx [hunk 2]
```diff
@@ -38,8 +38,7 @@ const connector = connect(mapStateToProps, mapDispatchToProps);
 
 const FormattedMessage = ({message}: FormattedMessageProps) => {
   if (message && message.content) {
-    const messageJSON = JSON.parse(message.content);
-    return <>{messageJSON.text}</>;
+    return <>{message.content.text}</>;
```
Comment (paulodiniz) [inline frontend/ui/src/pages/Inbox/ConversationListItem/index.tsx:41]: This will break with Twilio because `message.content` is a String

Comment (chrismatix) [reply frontend/ui/src/pages/Inbox/ConversationListItem/index.tsx:41]: I thought this was already using the render library to be honest.

**Review by chrismatix — APPROVED** [review]

**Pull request merged**